nextjs/nextjs-legacy/package.json

```diff
@@ -10,7 +10,8 @@
     "newrelic": "^12.0.0",
     "next": "^14.1.4",
     "pino": "^8.19.0",
-    "react": "^18.2.0"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
```

Comment:
Thank you for the PR, but I am not seeing any error mentioning "react-dom" when I follow the instructions in the Readme (`npm run build && npm start`).

Can you provide a minimal reproduction of the error you are encountering?

Reply:
@jsumners-nr Thank you for wonderful lib and reply. 
 I got same error when running script dev or build and not sure why you got no errors. Assuming "react-dom" installled globally in your environment?

### 📌 Reproduction 1:

1. yarn dev or npm run dev
2. visit localhost:3000

### 📌 Reproduction 2:

1. yarn build or npm run build

### 📌 Error Screenshot:

<img width="742" alt="스크린샷 2024-09-09 오후 8 44 55" src="https://github.com/user-attachments/assets/9d594258-adcf-4147-a9ff-836ebf5c9718">

Reply:
And I think it's wiered why you got no errors. Next.js requires "react-dom" and there is no "react-dom" in package.json.

<img width="742" alt="스크린샷 2024-09-09 오후 8 51 22" src="https://github.com/user-attachments/assets/c0547d4a-c99b-4ed5-a1eb-66ea16595234">

Reply:
> Assuming "react-dom" installled globally in your environment?

No. It is a dependency of `next`:

```
❯ npm ls react-dom
nextjs-legacy@ /Users/jsumners/Projects/team-repos/newrelic-node-examples/nextjs-legacy
└─┬ next@14.2.8
  └── react-dom@18.3.1
```

Running `npm install` should result in the correct dependencies being installed. Please remove the `node_modules` directory and run `npm install`.

Reply:
Thank you for reply. It is weired that when i run this project with `npm` it works. However, when i run it with `yarn` it requires react-dom.

My yarn version: 1.22.19

How about mention at readme.md that `npm` package manager is suggested to use?

Reply:
Yeap. I will close this pr and make new one.